galata/test/jupyterlab/notebook-search.test.ts

```diff
@@ -161,6 +161,34 @@ test.describe('Notebook Search', () => {
     await expect(page.locator('.jp-DocumentSearch-overlay')).toBeVisible();
   });
 
+  test('Populate search box with text selected in rendered markdown', async ({
+    page
+  }) => {
+    // Render the markdown cell to be able to select text from it.
+    await page.notebook.runCell(1, true);
+    const cell = await page.notebook.getCellLocator(1);
+
+    // Select a word for example here using "JupyterLab" by double clicking on it.
+    await cell!.getByText('JupyterLab').dblclick();
```

Comment:
@MUFFANUJ I think the test is timing out here because the markdown cell indeed does not have "JupyterLab" text in it:

<img width="952" height="495" alt="image" src="https://github.com/user-attachments/assets/00b7076a-b815-4d2a-b544-822efb1975a4" />